force-app/main/default/lwc/attendance/attendance.js

```diff
@@ -85,6 +123,8 @@ export default class Attendance extends LightningElement {
         } else if (result.error) {
             handleError(result.error);
```

Comment:
Not part of this work item but I thought we were doing a console.log here since this happens on load and we don't like to toast an error on load?